pkg/generate/cspc.go

```diff
@@ -181,10 +182,14 @@ func makePools(poolType string, nDevices int, bd map[string][]v1alpha1.BlockDevi
 			})
 		}
 		return &spec, nil
-	} else if poolType == string(cstorv1.PoolMirrored) {
-		if nDevices%2 != 0 {
-			return nil, fmt.Errorf("mirrored pool requires multiples of two block device")
+	case string(cstorv1.PoolMirrored), string(cstorv1.PoolRaidz), string(cstorv1.PoolRaidz2):
+		// 1. Check for devices per node count against the fixed minimum
+		min := minCount()[poolType]
+		if nDevices < min {
```

Comment:
Currently there's a check for minimum number(n=1) of BDs in a RaidGroup, more inputs will be needed to pass the webhook validation of `2^n + 1` & `2^n + 2` BDs in a RaidGroup.